### PR TITLE
Anomali threat stream v3 changes

### PR DESCRIPTION
**Pull request by ilappe**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/30158

## Minimum version of Cortex XSOAR
- [ ] 5.5.0

## Does it break backward compatibility?
   - [x] Yes
       - as discussed with @Arsenikr in https://github.com/demisto/etc/issues/30158

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/Anomali_ThreatStream/Integrations/Anomali_ThreatStream_v2/Anomali_ThreatStream_v2.py [hunk 1]
```diff
@@ -433,44 +463,60 @@ def test_module(client: Client):
     demisto.results('ok')
 
 
-def ips_reputation_command(client: Client, ip, threshold=None, status="active,inactive"):
+def ips_reputation_command(client: Client, ip, threshold=None, status="active"):
```
Comment (teizenman) [inline Packs/Anomali_ThreatStream/Integrations/Anomali_ThreatStream_v2/Anomali_ThreatStream_v2.py:466]: You moved the status input to the params, and yet you only send the args to these commands.

**Review by teizenman — CHANGES_REQUESTED** [review]

File: Packs/Anomali_ThreatStream/Integrations/Anomali_ThreatStream_v2/Anomali_ThreatStream_v2.py [hunk 2]
```diff
@@ -1,19 +1,18 @@
...
-import requests
-from requests.exceptions import MissingSchema, ConnectionError
 
 # Disable insecure warnings
 requests.packages.urllib3.disable_warnings()
 
 ''' GLOBALS/PARAMS '''
 
-VENDOR_NAME = 'ThreatStream'
+THREAT_STREAM = 'ThreatStream'
```
[the remark this answers is not in the document]
Comment (ilappe) [reply Packs/Anomali_ThreatStream/Integrations/Anomali_ThreatStream_v2/Anomali_ThreatStream_v2.py:13]: It was a conflict in merge from master
when i start implement it wasn't exist and i use it in lot of places so i replace it

File: Packs/Anomali_ThreatStream/Integrations/Anomali_ThreatStream_v2/Anomali_ThreatStream_v2.py [hunk 3]
```diff
@@ -1119,13 +1108,20 @@ def main():
...
-
     CREDENTIALS['username'] = user_name
     CREDENTIALS['api_key'] = api_key
 
+    # init threshold from params
+    THRESHOLDS_FROM_PARAM['url'] = arg_to_number(params.get('url_threshold'))
+    THRESHOLDS_FROM_PARAM['ip'] = arg_to_number(params.get('ip_threshold'))
+    THRESHOLDS_FROM_PARAM['file'] = arg_to_number(params.get('file_threshold'))
+    THRESHOLDS_FROM_PARAM['domain'] = arg_to_number(params.get('domain_threshold'))
+    THRESHOLDS_FROM_PARAM['threatstream-email-reputation'] = arg_to_number(params.get('email_threshold'))
```
Comment (teizenman) [inline Packs/Anomali_ThreatStream/Integrations/Anomali_ThreatStream_v2/Anomali_ThreatStream_v2.py:1123]:
```suggestion
    THRESHOLDS_FROM_PARAM = {
         'url' = arg_to_number(params.get('url_threshold')),
         'ip' = arg_to_number(params.get('ip_threshold')),
         'file' = arg_to_number(params.get('file_threshold')),
         'domain' = arg_to_number(params.get('domain_threshold')),
         'threatstream-email-reputation' = arg_to_number(params.get('email_threshold')),
    }
```

**Review by teizenman — CHANGES_REQUESTED** [review]
Please set up a demo.

File: Packs/Anomali_ThreatStream/Integrations/Anomali_ThreatStream_v2/Anomali_ThreatStream_v2.py (hunk 3 above)
Comment (ilappe) [reply Packs/Anomali_ThreatStream/Integrations/Anomali_ThreatStream_v2/Anomali_ThreatStream_v2.py:1123]: Are you sure?
it a global param and to init it as suggested we need to declare it 
`global THRESHOLDS_FROM_PARAM `
from my point of view the current implementation is more intuitive and clean

Comment (teizenman) [reply Packs/Anomali_ThreatStream/Integrations/Anomali_ThreatStream_v2/Anomali_ThreatStream_v2.py:1123]: It is up to you to decide which way to go here.
I have full faith in your decision.

**Pull request closed without merging**